learning/tour-of-beam/frontend/lib/pages/tour/state.dart

```diff
@@ -191,7 +209,23 @@ class TourNotifier extends ChangeNotifier with PageStateMixin<void> {
     if (content == null) {
       return;
     }
+
+    await _checkHasSavedSnippet();
+    _setSavedTypeIfHas();
     await _setCurrentSnippet();
+    notifyListeners();
+  }
+
+  Future<void> _checkHasSavedSnippet() async {
```

Comment:
'check' should be avoided because it does not answer what it does in either case.
Something like `_loadUserSnippetIdIfNot()`?